src/js/mixins/evented.js

```diff
@@ -0,0 +1,385 @@
+/**
+ * @file mixins/evented.js
```

Comment:
this should have a module definition as well so that it will show up correctly on the website. `@module evented`